src/components/CMessageRow.vue

```diff
@@ -70,8 +78,7 @@ export default {
       canCancel: true,
       isOpened: false,
       onError: false,
-      error: '',
-      txtBtnCancel: 'Cancel'
+      error: ''
```

Comment:
what is this error ? I think you should find a way to display a message if there is an error on requeue (example: if it is part of a pipeline).

Reply:
@antoinerabany  okay, let me fix it